PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+# My Awesome Pull Request
+
+## What does this change?
+
+## What does success look like and how do you plan to measure success?
+
+## Does this affect other platforms - Amp, Apps, etc?
+
+## Screenshots
```

Comment:
It might also be worth having an 'Request for comment' field - where the developer @mentions people who might have specific feedback, and where they can pre-empt certain questions about their approach and design.